firewood/src/merkle/node.rs

```diff
@@ -243,15 +243,19 @@ impl LeafNode {
 }
 
 #[derive(PartialEq, Eq, Clone)]
-pub struct ExtNode(
-    pub(super) PartialPath,
-    pub(super) DiskAddress,
-    pub(super) Option<Vec<u8>>,
-);
+pub struct ExtNode {
+    path: PartialPath,
+    chd: DiskAddress,
+    chd_encoded: Option<Vec<u8>>,
+}
 
 impl Debug for ExtNode {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> Result<(), fmt::Error> {
-        write!(f, "[Extension {:?} {:?} {:?}]", self.0, self.1, self.2)
+        write!(
+            f,
+            "[Extension {:?} {:?} {:?}]",
+            self.path, self.chd, self.chd_encoded
```

Comment:
More of an FYI than comment or change-request:
To prevent forgetting to update implementations like these, it's best to destructure the fields such that you get a compiler error if you add a field. 
```suggestion
        let Self { path, chd, chd_encoded } = self;

        write!(
            f,
            "[Extension {path:?} {chd:?} {chd_encoded:?}]",
```